generator/generator_tool/generator_tool.cpp

```diff
@@ -178,10 +181,8 @@ DEFINE_string(popular_places_data, "",
               "Input Popular Places source file name. Needed both for World intermediate features "
               "generation (2nd pass for World) and popular places section generation (5th pass for "
               "countries).");
-DEFINE_string(brands_data, "",
-              "Path to json with OSM objects to brand ID map.");
-DEFINE_string(brands_translations_data, "",
-              "Path to json with brands translations and synonyms.");
+DEFINE_string(brands_data, "", "Path to json with OSM objects to brand ID map.");
```

Comment:
Please factor out formatting changes to a separate PR next time.
I don't see how brands_data is relevant to this PR.

Reply:
My fault. I will change my workflow to format only my changes, not the complete file